src/security-server/system-test/src/intTest/java/org/niis/xroad/ss/test/globalconf/glue/GlobalConfStepDefs.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2019- Nordic Institute for Interoperability Solutions (NIIS)
+ * Copyright (c) 2018 Estonian Information System Authority (RIA),
+ * Nordic Institute for Interoperability Solutions (NIIS), Population Register Centre (VRK)
+ * Copyright (c) 2015-2017 Estonian Information System Authority (RIA), Population Register Centre (VRK)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.niis.xroad.ss.test.globalconf.glue;
+
+import ee.ria.xroad.common.conf.globalconf.ConfigurationPartMetadata;
+
+import com.codeborne.selenide.Selenide;
+import com.nortal.test.testcontainers.TestContainerService;
+import io.cucumber.java.en.Step;
+import org.niis.xroad.ss.test.ui.container.ReverseProxyAuxiliaryContainer;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.testcontainers.utility.MountableFile;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class GlobalConfStepDefs {
+
+    @Autowired
+    ReverseProxyAuxiliaryContainer reverseProxyAuxiliaryContainer;
+
+    @Autowired
+    TestContainerService testContainerService;
+
+    @SuppressWarnings("checkstyle:MagicNumber")
+    @Step("Central Server's global conf is updated by a new active signing key")
+    public void centralServersGlobalConfSignKeysAreRotated() {
+        var newGlobalConfFiles = MountableFile.forClasspathResource("files/global_conf_signed_with_rotated_keys");
+        reverseProxyAuxiliaryContainer.getTestContainer().copyFileToContainer(newGlobalConfFiles, "/var/lib/xroad/public");
+
+        // As Security server polls for global conf every 3 secs, ensure that SS has the new global conf loaded
+        Selenide.sleep(4000);
```

Comment:
is 1sec gap enough for slow instances like CI?

Reply:
Initial (also slowest) configuration download took ~0.25 secs on my local virtualized docker environment.
CI instances have weaker specs true, but they're also linux meaning they run on native docker.
1 sec buffer should be enough in my estimation, but we could go with 2 secs just to be sure.